RollGen.Domain/DomainDice.cs

```diff
@@ -49,7 +49,7 @@ public string ReplaceWrappedExpressions<T>(string source, string expressionOpen
                 var lenientReplacedDice = ReplaceDiceExpression(matchGroupValue, true);
                 var unevaluatedMatch = Evaluate(lenientReplacedDice);
                 var target = expressionOpen + matchGroupValue + expressionClose;
-                var replacement = BooleanOrType<T>(unevaluatedMatch).ToString();
+                var replacement = Utils.BooleanOrType<T>(unevaluatedMatch).ToString();
```

Comment:
Also, instead of a static class, can we make this injectable?  Like a "converter" or something.